lib/rubocop/cop/style/yoda_condition.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for Yoda conditions where the two parts of an expression
+      # are reversed from the typical order in a conditional statement.
```

Comment:
Sounds reasonable to me. Similar wording can be used in the style guide I guess.